fail to build:  error: package android.support.test.rule does not exist

Why I try to build the examples, it fails due to  `android.support.test.rule not found`:

```bash
$ ./gradlew  examples:ex2_espresso:assembleDebug examples:ex4_uiautomator:assembleDebug  

> Task :examples:ex2_espresso:compileSnippetDebugJavaWithJavac FAILED
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/Espresso.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/assertion/ViewAssertions.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/matcher/ViewMatchers.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/action/ViewActions.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
/Users/darren/Sources/go/src/github.com/google/mobly-snippet-lib/examples/ex2_espresso/src/snippet/java/com/google/android/mobly/snippet/example2/EspressoSnippet.java:25: error: package android.support.test.rule does not exist
import android.support.test.rule.ActivityTestRule;
                                ^
/Users/darren/Sources/go/src/github.com/google/mobly-snippet-lib/examples/ex2_espresso/src/snippet/java/com/google/android/mobly/snippet/example2/EspressoSnippet.java:32: error: cannot find symbol
    public ActivityTestRule<MainActivity> mActivityRule =
           ^
  symbol:   class ActivityTestRule
  location: class EspressoSnippet
/Users/darren/Sources/go/src/github.com/google/mobly-snippet-lib/examples/ex2_espresso/src/snippet/java/com/google/android/mobly/snippet/example2/EspressoSnippet.java:33: error: cannot find symbol
            new ActivityTestRule<>(MainActivity.class);
                ^
  symbol:   class ActivityTestRule
  location: class EspressoSnippet
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/ViewAssertion.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/ViewInteraction.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
/Users/darren/.gradle/caches/transforms-1/files-1.1/espresso-core-3.0.2.aar/c05bd1847e045a233c593a0c4bca711a/jars/classes.jar(android/support/test/espresso/ViewAction.class): warning: [classfile] MethodParameters attribute introduced in version 52.0 class files is ignored in version 51.0 class files
3 errors
7 warnings
```

full debug output attached as file [debug.log](https://github.com/google/mobly-snippet-lib/files/2120301/debug.log)

Anything wrong with my environment setup?

--- a/examples/ex2_espresso/build.gradle
+++ b/examples/ex2_espresso/build.gradle
@@ -39,7 +39,6 @@ dependencies {
     // codebase.
     androidTestImplementation 'com.android.support:support-annotations:27.1.1'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test:rules:1.0.2'
     androidTestImplementation('com.android.support.test.espresso:espresso-core:3.0.1', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
@@ -50,6 +49,7 @@ dependencies {
     //snippetCompile 'com.google.android.mobly:mobly-snippet-lib:1.3.0'
     snippetImplementation project(':mobly-snippet-lib')
 
-    snippetImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     snippetImplementation 'com.android.support:support-annotations:27.1.1'
+    snippetImplementation 'com.android.support.test:rules:1.0.2'
+    snippetImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 }

